aiohttp/client_reqrep.py

```diff
@@ -880,6 +881,10 @@ async def start(self, connection: "Connection") -> "ClientResponse":
                 self.cookies.load(hdr)
             except CookieError as exc:
                 client_logger.warning("Can not load response cookies: %s", exc)
+
+        # certificate
+        if self._connection is not None and self._connection.transport is not None:
```

Comment:
Labeling complex conditions improves maintainability:
```suggestion
        is_tls_transport = self._connection is not None and self._connection.transport is not None:
        if is_tls_transport:
```